src/Listbox.js

```diff
@@ -119,7 +120,7 @@ export const ListboxList = {
                 this.context.close()
                 break
               case 'Tab':
-                e.preventDefault()
+                this.context.close()
```

Comment:
This was done on purpose, because `Tab` inside an open `Listbox` should move focus to the next item after the `ListboxButton`.

Without preventDefault the `Listbox` is closed which sets focus on the `ListboxButton`. Next the default `Tab` behavior triggers and sets focus to the next focusable element after the button.

That said, I forgot to account for `Shift` + `Tab`. In that case focus should actually be put on the `ListboxButton`, so we need `preventDefault` there.